src/repo/index.js

```diff
@@ -16,6 +16,7 @@ const createDB = async (server, {
     GKB_DB_NAME,
     GKB_DBS_PASS,
     GKB_DBS_USER,
+    GKB_DB_PASS,
```

Comment:
Minor. Since this second argument is an object, we could put these properties in alphabetic order.

Reply:
done